features-json/gamepad.json

```diff
@@ -219,10 +219,27 @@
       "30":"n"
     },
     "and_chr":{
-      "44":"n"
+      "36":"n",
+      "37":"y",
+      "38":"y",
+      "39":"y",
+      "40":"y",
+      "41":"y",
+      "42":"y",
+      "43":"y",
```

Comment:
Data for older versions of Android Chrome & Android Firefox aren't currently accepted. See the Travis build failure.